static/gsApp/views/amCheckout/steps/setBudgetAndReserves.tsx

```diff
@@ -1,227 +0,0 @@
-import {Component} from 'react';
```

Comment:
Full circle moment to remove a component you created 😀

Reply:
do i laugh or do i cry 😂